openedx_events/learning/data.py

```diff
@@ -589,3 +589,19 @@ class BadgeData:
     uuid = attr.ib(type=str)
     user = attr.ib(type=UserData)
     template = attr.ib(type=BadgeTemplateData)
+
+
+@attr.s(frozen=True)
+class VerificationAttemptData:
+    """
+    Attributes defined for the persona-integration IDV attempt data object.
+
+    Arguments:
+        asdf (type): description
+    """
+
+    user = attr.ib(type=UserData)
+    name = attr.ib(type=str)
```

Comment:
`name` and `expiration_date` are both optional, so I think a `default=None` would be sensible here.